### ISSUE TITLE
handling chromosome X

### ISSUE THREAD

**Issue by xiaolinzhu** [KChen-lab/Monopogen#73]
Hi there, can monopogen handle chromosome X, especially in males? Is there a standard of practice? Thanks! -Xiaolin

**Comment by jinzhuangdou** [comment]
No, Monopogen does not work on chrX since the imputation is a bit different from autosome.

**Comment by jinzhuangdou** [comment]
We do not have such experience on chrX imputation now.
